language/fr/permissions_ajax_shoutbox.php

```diff
@@ -35,7 +35,7 @@
 // in a url you again do not need to specify an order e.g., 'Click %sHERE%s' is fine
 //
 // Some characters you may want to copy&paste:
-// ’ » “ ” …
+// ’ « » “ ” …
```

Comment:
Hi paul999,

For french people, `« »` is same as `“ ”`
And the chars `«` is not present on our French keyboard. That's why we added this chars on this line.

So... why request to remove it ?
The line before says `
 // Some characters you may want to copy&paste:` and the copy/paste is the only way to use this chars easily.

Regards.